frontend/src/lib/parseSearchQueries.ts

```diff
@@ -0,0 +1,8 @@
+export function parseSearchQueries(value: string): string[] {
+  const queries = value
+    .split(/[\s,]+/) // split on whitespaces and commas
+    .map((query) => query.trim())
+    .filter(Boolean); // remove empty strings (e.g. when `value` is `123,,456`)
```

Comment:
i would avoid doing "tricks" like this.  instead, use lodash library which provides nicely named methods for these operations.   e.g. compact (removes falsy values) and uniq which will make the list unique.

Reply:
Thanks, @alisman. I'll look into these methods. Considering that this PR has been merged, I'll incorporate your feedback in a separate PR.

Reply:
yeah, this is not too important since what you have works.  but it's good to get in the habit of avoiding these kind of things because it's hard to read.

Reply:
That makes sense. Conveniently, I'm currently working on a PR that modifies this function, so I'll incorporate your feedback there and add you as another reviewer. Thanks again.